### ISSUE TITLE
livecd-iso-to-disk fails to create liveusb for Fedora Workstation 37 Beta iso

### ISSUE THREAD

**Issue by laolux** [livecd-tools/livecd-tools#253]
I am using Fedora 36 to create a liveusb for Fedora Workstation 37 Beta.
As root I run `livecd-iso-to-disk --format ext4 /home/user/Fedora-Workstation-Live-x86_64-37_Beta-1.5.iso /dev/sdb`

This fails with

    Source image is '/home/user/Fedora-Workstation-Live-x86_64-37_Beta-1.5.iso'
    Verifying image...
    /home/user/Fedora-Workstation-Live-x86_64-37_Beta-1.5.iso:   3a59d1f243057ea068d29e10a68274a1
    Fragment sums: 53a7a56ca6ffed258e561cccb19923611f71469e4975fa15b755eeb61498
    Fragment count: 20
    Supported ISO: no
    Press [Esc] to abort check.
    Checking: 100.0%

    The media check is complete, the result is: PASS.

    It is OK to use this media.
    Error 1 detected at line 2028 /usr/bin/livecd-iso-to-disk
    Error 1 detected at line 2028 /usr/bin/livecd-iso-to-disk

The error does not appear when I use an image for Fedora Workstation 36 Beta. In this case the tool seems to work fine.

Note: The issue seems related to #244

**Comment by FGrose** [comment]
Yes, the .iso image structure has changed with Fedora 37.  A new version of `livecd-iso-to-disk` will be required. I'm testing that now.

**Comment by laolux** [comment]
Great that you work on this already! I would be happy to help testing, either now or when the package hits bodhi.

**Comment by mahdiaqallal** [comment]
Hi @FGrose 

Would the next version of `livecd-iso-to-disk` you're working on, fix the #244 ? 

My contribution: The later issue seems to specific to the .iso files of Silverblue since at least release 35. (I've reproduces it for releases 35, 36). For Silverblue 37 I will test after it's final release date in a couple of weeks )

The only workaround I've found to far is to revert to livecd-tools-27.1 that was  included in Fedora 34 but now EOL

**Comment by Naim-Bijapure** [comment]
getting same issue on fedora 37 iso .
is  older version of livecd-tools work with fedora 37 ?

**Comment by bioinfornatics** [comment]
same issue here,
As this bug is open since 1 year how people are installing fedora livecd iso to usb disk ?

**Comment by dreua** [comment]
> how people are installing fedora livecd iso to usb disk ?

[Fedora Media Writer](https://github.com/FedoraQt/MediaWriter)

**Comment by luisfl** [comment]
Same problem in FC38. The error is caused by the command

    lsinitrd /initrd*.img --unpack -f usr/lib/modules/*/kernel -v 2>&1

Namely, the variable CONFIG_SRC failed to be set. And the reason is simply that neither $SRCMNT/isolinux/ nor $SRCMNT/syslinux/ exist at that step in the livecd-iso-to-disk script. Actually, those forlders do not exist in $SRCMNT, at all.

/begin{rant}
Of course no responses from FC. I don't know why we still report bugs to Fedora. I don't remember the last time one of my bug reports was attended.
/end{rant}

**Comment by gbuloz** [comment]
I've open a Fedora bug report her last december : https://bugzilla.redhat.com/show_bug.cgi?id=2152842

Waiting for a fix for livecd-iso-to-disk, If you just want to burn your live/install ISO image to a USB stick where you can also store some data, I've created a simple live2usb.sh script. A GPT table with an EFI system partition and FAT32 read-write filesystem is used for that. Tried with Fedora 36 live xfce (ISO with DOS parition table) and Fedora 37 (ISO with GPT partition table).
[live2usb.sh.gz](https://github.com/livecd-tools/livecd-tools/files/11633891/live2usb.sh.gz)

**Comment by rokiden** [comment]
> I've created a simple live2usb.sh script

Thank you for inspiration. I made my own version with ext4 partition and persistence directory.
https://gist.github.com/rokiden/e3ba8c587a6475f959bd2ce09ff9dfd0
